src/quo/components/numbered_keyboard/numbered_keyboard/view.cljs

```diff
@@ -17,13 +17,15 @@
 
 (defn- view-internal
   []
-  (fn [{:keys [disabled? theme blur? left-action delete-key? on-press]}]
+  (fn [{:keys [disabled? theme blur? left-action delete-key? on-press on-delete]
+        :or   {left-action :none}}]
     [rn/view
      {:style style/container}
      (for [row-index (range 1 4)]
        ^{:key row-index}
        [rn/view {:style style/row-container}
         (for [column-index (range 1 4)]
+          ^{:key column-index}
```

Comment:
@ulisesmac - I think we need to be a bit more careful with keys right? like it should be something more unique 🤔

Reply:
@Rende11 -  the `column-index` would be the same for each row (React might throw warnings). 

We should add the `row-index` and `column-index` to maintain the uniqueness across the loop/keypad.

Reply:
@smohamedjavid
I'm guided by this doc -https://react.dev/learn/rendering-lists#rules-of-keys

> Keys must be unique among siblings. However, it’s okay to use the same keys for JSX nodes in different arrays.

Reply:
:thinking: I have had problems in the past when using just idx as keys. I'd call them using a more descriptive ID.

I thought the same as @smohamedjavid : react might throw warnings.

Reply:
Replaced with `^{:key (str row-index column-index)}`